daml-script/test/BUILD.bazel

```diff
@@ -121,3 +121,18 @@ client_server_test(
     server_files = ["$(rootpath :script-test.dar)"],
     tags = ["exclusive"],
 )
+
+sh_test(
```

Comment:
It would be nice to verify a bit more in this test, e.g., verify the expected output.

Reply:
Good point. I've added a failing test-case and made the test runner a bit more sophisticated. It now checks that the expected set of tests succeeds or fails.

This also revealed a race-condition in the test-runner with the in-memory ledger. If a test-case failed then the flow was torn down causing the sandbox to be shut-down which caused other still running test-cases to fail.

I've changed the test-runner so that the flow is not torn down on test-case failure, but instead all tests are run to completion. The final exit-code tells if any test-case failed.